services/system/Accounts/plugin/accounts/src/interfaces/admin.rs

```diff
@@ -50,15 +42,7 @@ impl Admin for AccountsPlugin {
 
```

Comment:
Oh, whoops - this is a minor bug introduced in a past pr
```
TransactAuthApi::get_query_token(&Client::my_service_account(), &user).unwrap();
```

That should be 
```
TransactAuthApi::get_query_token(&app.app.unwrap(), &user).unwrap();
```

(Or, after [this merge](https://github.com/gofractally/psibase/pull/1345):
```
TransactAuthApi::get_query_token(&app, &user).unwrap();
```
)